src/gui/shell.cpp

```diff
@@ -523,7 +523,25 @@ void Shell::handleRedraw(const QByteArray& name, const QVariantList& opargs)
 		m_pum.hide();
 	} else if (name == "mode_info_set") {
 		// TODO
-	} else if (name == "default_colors_set") {
+        } else if (name == "default_colors_set") {
+
+        	//"default_colors_set" have 5 parameters
+        	// 0 - foreground (default = -1)
+        	// 1 - background (default = -1)
+        	//...
+
+        	//don't handle default 
```

Comment:
We can probably safely assume the last 2 parameters will never be used; but is it safe  to ignore the whole default_colors_set call if *either* fg or bg is `-1` ?

Reply:
> We can probably safely assume the last 2 parameters will never be used; but is it safe to ignore the whole default_colors_set call if _either_ fg or bg is `-1` ?

Yep. We can ignore the whole "default_colors_set" because our colorscheme using always 2 colors (for fg and bg). We cant set a new theme without 2 colors.